### ISSUE TITLE
Create a tool for initialize a MAS.

### ISSUE THREAD

**Issue by gallandarakhneorg** [sarl/sarl#401]
During my stay abroad, I'm working with Transport Science researchers that are needing to have a multiagent system already spawned (with agents, inner agents, and spaces) at start-up.

The description of the system to spawn of the system purely in SARL is possible.
But it seems a simple description file may help them for starting, such as an the following XML file.
This may help SARL users to specify the initial state of an holarchy.
```xml
<mas>
    <spaces>
        <space id="SPID1" type="EventSpace">
        <space id="SPID2" type="EventSpace">
    <spaces>
    <agent type="MyAgent" id="ID1">
        <joins>
            <space id="SPID1">
        <joins>
        <spaces>
            <space id="SPID2">
        <spaces>
        <agent type="MyAgent" id="ID1">
            <joins>
                <space id="SPID2">
            <joins>
        </agent>
    </agent>
</mas>
```

We should decide if this problem is at the level of the system specification (with SARL), or at the runtime level (with Janus).
For now, I consider this problem more related to the platform.
What do you think?

**Comment by ngaud** [comment]
My opinion: it is not related to SARL but only an additional feature for Janus.
A simple parser of the XML that will creates the corresponding elements.

That could also be part of the Janus deployment module: in charge of managing the deployment of a whole SARL-based application on a given network node. In this case, we must have a more complete description of the deployment configuration and XML is probably not the best choice for describing this kind of configuration.

**Comment by srodriguez** [comment]
I agree with Nicolas that we should look into the deployment configuration, but I think it should be a side project to configure Janus deployment configurations. IMHO, this means more than just spawning agents, it also involves starting Janus Kernels (even in different nodes), etc.
There are a lot of large projects we could use.

**Comment by gallandarakhneorg** [comment]
I agree with your comments.
At the end we should give a tutorial for doing that.
It seems to be mandatory in cases where the SARL users wants to spawn 200000 agents with specific organizational schemes.
